File: python/fizzbuzz.py
def fizzbuzz(n):
    if n == 0:
        return n
    elif (n % 3 == 0 and n % 5 == 0):
        return 'FizzBuzz'
    elif n % 3 == 0:
        return 'Fizz'
    elif n % 5 == 0:
        return 'Buzz'
    else:
        return n

File: ruby/fizzbuzz.rb
def fizzbuzz(number)
  return number if number.zero?

  if (number % 3).zero? && (number % 5).zero?
    'FizzBuzz'
  elsif (number % 3).zero?
    'Fizz'
  elsif (number % 5).zero?
    'Buzz'
  else
    number
  end
end
